NiketaPopatChaudhari/serverless-azure-functions
# added nodejsv20 support to azure function

What did you implement:
added nodejsv20 support to azure function

How did you implement it:
azure function will become available to use the nodejs20 runtime.
updated the available runtimes at src/config/runtime.ts

How can we verify it:
Can you generate an azure function with serverless-framework and deploy it with nodejs20 runtime.
Ex: runtime: nodejs20

Todos:
Note: Run npm run test:ci to run all validation checks on proposed changes

Ensure there are no lint errors.
Validate via npm run lint
Note: Some reported issues can be automatically fixed by running npm run lint:fix
Write tests and confirm existing functionality is not broken.
Validate via npm test

Is this ready for review?: YES
Is it a breaking change?: NO
